src/main/groovy/com/anotherchrisberry/spock/extensions/retry/RetrySpecExtension.groovy

```diff
@@ -44,7 +46,25 @@ class RetrySpecExtension extends AbstractAnnotationDrivenExtension<RetryOnFailur
     }
     double getDelaySeconds(RetryOnFailure retries) {
         String defaultDelay = Double.toString(retries.delaySeconds())
-        return Double.parseDouble(System.getProperty("spck-retry.delaySeconds", defaultDelay))
+        return Double.parseDouble(System.getProperty("spock-retry.delaySeconds", defaultDelay))
+    }
+
+    Predicate<Throwable> getPredicate(final RetryOnFailure retries) {
```

Comment:
Maybe rename to `getExceptionChecks` or something more descriptive about what the `Predicate` is for?